src/main/java/edu/cornell/kfs/sys/service/impl/DocumentMaintenanceServiceImpl.java

```diff
@@ -1,45 +1,100 @@
 package edu.cornell.kfs.sys.service.impl;
 
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.dataaccess.ActionItemNoteDetailDto;
 import edu.cornell.kfs.sys.dataaccess.DocumentMaintenanceDao;
 import edu.cornell.kfs.sys.service.DocumentMaintenanceService;
+
+import org.apache.commons.collections.CollectionUtils;

```

Comment:
This should use the collections4 version of CollectionUtils instead.

Reply:
good catch